logistics/handling-money.md

```diff
@@ -0,0 +1,24 @@
+# Handling Money
+
+RustBridge should be free for attendees.
```

Comment:
if you wanted to put line breaks here, they need to be separated by an empty line so markdown picks them up